randovania/games/fusion/logic_database/Sector 3 PYR.txt

```diff
@@ -392,7 +392,10 @@ Extra - room_id: [9]
   > Door to Connection to Sector 1 (SRX)
       All of the following:
           Screw Attack
-          Speed Booster or Can Climb High Jump Wall
+          Any of the following:
+              Speed Booster or Can Climb High Jump Wall
+              # https://youtu.be/7dIXfR4dkTA
```

Comment:
reminder for future videos: please include an input viewer :)
(also you might wanna avoid any kind of political imagery)